site/docs/plugins/commands.md

```diff
@@ -200,119 +200,115 @@ Let's say we want to have developer-only commands.
-├─ types.ts
-tsconfig.json
+.
+├── bot.ts
+├── types.ts
+└── commands/
+    ├── admin.ts
+    └── users/
+        ├── group.ts
+        ├── say-hello.ts
+        └── say-bye.ts
 ```
 
-The following code group exemplifies how we could implement a developer only command group, and update the Telegram client Command menu accordingly.
-Make sure you take notice of the different patterns being use in the `admin.ts` and `group.ts` file-tabs.
+The following code group exemplifies how we could implement a developer only command group, and update the Telegram client command menu accordingly.
+Make sure you take notice of the different patterns being used in the `admin.ts` and `group.ts` file-tabs.
 
 ::: code-group
 
-```ts [types.ts]
-export type MyContext = Context & CommandsFlavor<MyContext>;
-```
-
 ```ts [bot.ts]
 import { devCommands } from "./commands/admin.ts";
 import { userCommands } from "./commands/users/group.ts";
 import type { MyContext } from "./types.ts";
 
-export const bot = new Bot<MyContext>("MyBotToken");
+export const bot = new Bot<MyContext>(""); // <-- put your bot token between the "" (https://t.me/BotFather)
 
 bot.use(commands());
 
 bot.use(userCommands);
-bot.use(devCommands);
+bot.filter((ctx) => ctx.from?.id == /** Put your ID here **/)
+      .use(devCommands);
+```
+
+```ts [types.ts]
+import type { Context } from "grammy";
+
+export type MyContext = CommandsFlavor<Context>;
 ```
 
 ```ts [admin.ts]
-import { userCommands } from './users/group.ts'
-import type { MyContext } from '../types.ts'
-
-export const devCommands = new CommandGroup<MyContext>()
-
-devCommands.command('devlogin', 'Greetings', async (ctx, next) => {
-   if (ctx.from?.id === ctx.env.DEVELOPER_ID) {
-      await ctx.reply('Hi to me')
-      await ctx.setMyCommands(userCommands, devCommands)
-   } else {
-     await next()
-   }
-})
-
-devCommands.command('usercount', 'Greetings', async (ctx, next) => {
-   if (ctx.from?.id === ctx.env.DEVELOPER_ID) {
-      await ctx.reply(
-        `Active users: ${/** Your business logic */}`
-    )
-   } else {
-     await next()
-   }
-})
-
-devCommands.command('devlogout', 'Greetings', async (ctx, next) => {
-    if (ctx.from?.id === ctx.env.DEVELOPER_ID) {
-       await ctx.reply('Bye to me')
-       await ctx.setMyCommands(userCommands)
-   } else {
-     await next()
-   }
- })
+import { userCommands } from './users/group.ts';
+import type { MyContext } from '../types.ts';
+
+export const devCommands = new CommandGroup<MyContext>();
+
+devCommands.command("devlogin", "Set command menu to dev mode", async (ctx, next) => {
+  await ctx.reply("Hello, fellow developer! Are we having coffee today too?");
+  await ctx.setMyCommands(userCommands, devCommands);
+});
+
+devCommands.command("usercount", "Display user count", async (ctx, next) => {
+  await ctx.reply(`Total users: ${/** Your business logic */}`);
+});
+
+devCommands.command("devlogout", "Reset command menu to user-mode", async (ctx, next) => {
```

Comment:
```suggestion
devCommands.command("devlogout", "Turn command menu to normal mode", async (ctx, next) => {
```

@KnorpelSenf, @rojvv what about this suggestion?

Reply:
Let's not change the original at this point anymore. Feel free to open a new PQ so we can discuss it there.